.gitignore

```diff
@@ -16,3 +16,4 @@ CHANGELOG.md
 node_modules
 npm-debug.log
 yarn-error.log
+.betterer.cache
```

Comment:
Can this one live here, or we should use individual global ignores for this type of caching things? 🤔

Reply:
Since we are all going to use it locally, I think it makes sense to have it in the repo :ok_hand: